pkg/controller/applicationmonitoring/applicationmonitoring_controller.go

```diff
@@ -160,7 +157,32 @@ func (r *ReconcileApplicationMonitoring) Reconcile(request reconcile.Request) (r
 
 func (r *ReconcileApplicationMonitoring) reconcileConfig(cr *applicationmonitoringv1alpha1.ApplicationMonitoring) (reconcile.Result, error) {
 	log.Info("Phase: Reconciling Config")
-	return reconcile.Result{}, r.createOrUpdateAdditionalScrapeConfig(cr)
+	err := r.syncBlackboxTargets(cr)
+	return reconcile.Result{RequeueAfter: time.Second * ReconcilePauseSeconds}, err
+}
+
+func (r *ReconcileApplicationMonitoring) syncBlackboxTargets(cr *applicationmonitoringv1alpha1.ApplicationMonitoring) error {
+	log.Info("Phase: Reconciling Config syncBlackboxTargets")
+
+	previous := cr.Status.Bbthash
+
+	bbtList := common.GetBTConfig()
+	crbbtList := cr.Spec.BlackboxTargets
+	crName := cr.ObjectMeta.Name
+	bbtList.BTs[crName] = crbbtList
+
+	flatList := common.Flatten()
+	current := joinQuote(flatList)
+
+	check, hash := r.hasDatasourceChanged(previous, current)
```

Comment:
typo, instead of datasource, should be something like maybe hasBlackboxTargetsListChanged